src/diffusers/schedulers/scheduling_dpmsolver_singlestep.py

```diff
@@ -320,7 +320,7 @@ def set_timesteps(self, num_inference_steps: int, device: Union[str, torch.devic
 
         if not self.config.lower_order_final and num_inference_steps % self.config.solver_order != 0:
             logger.warn(
-                "Changing scheduler {self.config} to have `lower_order_final` set to True to handle uneven amount of inference steps. Please make sure to always use an even number of `num_inference steps when using `lower_order_final=True`."
+                "Changing scheduler {self.config} to have `lower_order_final` set to True to handle uneven amount of inference steps. Please make sure to always use an even number of `num_inference steps when using `lower_order_final=False`."
```

Comment:
I think it's a typo here but let me know if it's not @patrickvonplaten 
from this PR https://github.com/huggingface/diffusers/pull/3413

Reply:
Yes LGTM - thanks!